openreview/tools.py

```diff
@@ -826,16 +836,21 @@ def assign(client, paper_number, conference,
 
     For example: passing in a reviewer to remove AND a reviewer to add should replace the first user with the second.
     '''
+    changed_groups = []
+    user = ""
+
     if reviewer_to_remove:
-        remove_assignment(client, paper_number, conference, reviewer_to_remove,
+        user, changed_groups = remove_assignment(client, paper_number, conference, reviewer_to_remove,
                           parent_group_params, parent_label, individual_label)
+
     if reviewer_to_add:
-        add_assignment(client, paper_number, conference, reviewer_to_add,
+        user, changed_groups = add_assignment(client, paper_number, conference, reviewer_to_add,
```

Comment:
If you have a `reviewer_to_remove` and a `reviewer_to_add`, I think `changed_groups` will only contain the result of `add_assignment`, right?

Reply:
Did you mean to say "If you have a reviewer_to_remove OR a reviewer_to_add..."? I am asking because we should not get both at the same time.
Also, "user" will contain the profile id, if available, while reviewer_to_add or reviewer_to_remove may contain email ids.

Reply:
I would like to have two different methods, one to add and another to remove. It is confusing to have both functionalities in a single place.

Reply:
@mspector Sorry, somehow I was under the assumption that this script does either reviewer addition or deletion at a time. I will fix this.

Reply:
We could mark 'assign' function as deprecated and use add_assignment and remove_assignment from the scripts.

Reply:
You're right, that docstring is ambiguous. I like Melisa's suggestion.

Reply:
Done. Please take a look @mspector and @melisabok.